src/plugin/action/join.sjs

```diff
@@ -11,13 +11,25 @@ module.exports = function (client, rawf, emitter) {
                 reject(new Error("No channel given to join action."));
                 return;
             }
+            var info = channel;
+             if (typeof(channel) === 'object'){
+                var joinInfo = {
+                    names: [],
+                    channel: channel.channel,
+                    nickname: client.nickname(),
+                    password: channel.password,
```

Comment:
I don't think we want the channel password to be in the joinInfo. I could be convinced otherwise.